src/components/__tests__/change-view.test.jsx

```diff
@@ -18,17 +22,25 @@ jest.mock('../../scripts/layered-storage', () => ({
   }
 }));
 
+// Change string values to date objects so they're parsed correctly
+simplePage.versions.forEach(version => {
+  version.capture_time = new Date(version.capture_time);
+});
+
 const mockChange = {
-  uuid: 'adc70e01-0723-4a28-a8e4-ca4b551ed2ae..0772dc83-7966-4881-8901-eceb051b9536',
-  uuid_from: 'adc70e01-0723-4a28-a8e4-ca4b551ed2ae',
-  uuid_to: '0772dc83-7966-4881-8901-eceb051b9536',
+  uuid: '70aa8b7b-d485-48dc-8295-7635ff04dc5b..ed4aed23-424e-4326-ac75-c3ad72797bad',
+  uuid_from: '70aa8b7b-d485-48dc-8295-7635ff04dc5b',
+  uuid_to: 'ed4aed23-424e-4326-ac75-c3ad72797bad',
```

Comment:
The previous IDs were not in the actual mock data for pages/versions. This wasn’t an issue with shallow rendering in Enzyme, but RTL doesn’t really have shallow rendering, so this needs to match up with the other mock data in order for it to work now (you could mock all the sub-components to alleviate this problem, but I think that’s not usually a great solution).